docs/resources/obs_bucket.md

```diff
@@ -1,352 +1,352 @@
+}
+```
+
+### Using CORS
+
+```hcl
+resource "huaweicloud_obs_bucket" "b" {
+  bucket = "obs-website-test.hashicorp.com"
+  acl    = "public-read"
+
+  cors_rule {
+    allowed_origins = ["https://obs-website-test.hashicorp.com"]
+    allowed_methods = ["PUT", "POST"]
+    allowed_headers = ["*"]
+    expose_headers  = ["ETag"]
+    max_age_seconds = 3000
+  }
+}
+```
+
+### Using object lifecycle
+
+```hcl
+resource "huaweicloud_obs_bucket" "bucket" {
+  bucket     = "my-bucket"
+  acl        = "private"
+  versioning = true
+
+  lifecycle_rule {
+    name    = "log"
+    prefix  = "log/"
+    enabled = true
+
+    expiration {
+      days = 365
+    }
+    transition {
+      days          = 60
+      storage_class = "WARM"
+    }
+    transition {
+      days          = 180
+      storage_class = "COLD"
+    }
+  }
+
+  lifecycle_rule {
+    name    = "tmp"
+    prefix  = "tmp/"
+    enabled = true
+
+    noncurrent_version_expiration {
+      days = 180
+    }
+    noncurrent_version_transition {
+      days          = 30
+      storage_class = "WARM"
+    }
+    noncurrent_version_transition {
+      days          = 60
+      storage_class = "COLD"
+    }
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `bucket` - (Required, String, ForceNew) Specifies the name of the bucket. Changing this parameter will create a new
+  resource. A bucket must be named according to the globally applied DNS naming regulations as follows:
+    + The name must be globally unique in OBS.
+    + The name must contain 3 to 63 characters. Only lowercase letters, digits, hyphens (-), and periods (.) are
+      allowed.
+    + The name cannot start or end with a period (.) or hyphen (-), and cannot contain two consecutive periods (.) or
+      contain a period (.) and a hyphen (-) adjacent to each other.
+    + The name cannot be an IP address.
+    + If the name contains any periods (.), a security certificate verification message may appear when you access the
+      bucket or its objects by entering a domain name.
+
+* `storage_class` - (Optional, String) Specifies the storage class of the bucket. OBS provides three storage classes:
+  "STANDARD", "WARM" (Infrequent Access) and "COLD" (Archive). Defaults to `STANDARD`.
+
+* `acl` - (Optional, String) Specifies the ACL policy for a bucket. The predefined common policies are as follows:
+  "private", "public-read", "public-read-write" and "log-delivery-write". Defaults to `private`.
+
+* `policy` - (Optional, String) Specifies the text of the bucket policy in JSON format. For more information about obs
+  format bucket policy,
+  see the [Developer Guide](https://support.huaweicloud.com/intl/en-us/perms-cfg-obs/obs_40_0004.html).
+
+* `policy_format` - (Optional, String) Specifies the policy format, the supported values are *obs* and *s3*. Defaults
+  to *obs*.
+
+* `tags` - (Optional, Map) A mapping of tags to assign to the bucket. Each tag is represented by one key-value pair.

```

Comment:
the description format should be start with Specifies, so that it is consistent with other param